### PR TITLE
Marlowe tutorial april2020

### PR DESCRIPTION
**Pull request by simonjohnthompson**
Updated section 10 of the tutorial for the new version of the playground.

### CODE REVIEW

File: marlowe-tutorial/doc/playground-overview.adoc [hunk 1]
```diff
@@ -17,130 +18,99 @@ contract in the browser-based tool, the Marlowe Playground, a web tool
...
-image:step4.png[Simulation step 4]
+=== Analysing a contract
 
-At any point it is possible to _undo_ a step by selecting the *Undo*
-button. From the current state, doing this repeatedly will return us to the
-position where Alice and Bob are prompted for a choice. If Alice and Bob now make _different_ choices, we reach the following situation, where Carol is asked to choose `0` (for pay) or `1` (for refund):
+The static analysis of a contract is performed by selecting the *Static analysis* tab in the pane at the foot of the page.
 
-image:step5.png[Simulation step 5]
+image:static-analysis.png[Static analysis]
```
Comment (palas) [inline marlowe-tutorial/doc/playground-overview.adoc:109]: It seems this image is missing, maybe it is not added to the repository.

File: marlowe-tutorial/doc/playground-overview.adoc [hunk 2]
```diff
@@ -2,9 +2,10 @@
 == The Marlowe Playground
 
 This tutorial gives an overview of the Marlowe Playground, an online
-tool that allows users to interact with and simulate the operation of
-embedded Marlowe contracts. It is available at
-{marloweplayground}.
+tool that allows users to create, to analyse, to interact with and to simulate the 
+operation of
+embedded Marlowe contracts. It is available at https://alpha.marlowe.iohkdev.io/[https://alpha.marlowe.iohkdev.io/]
```
[the remark this answers is not in the document]
Comment (simonjohnthompson) [reply marlowe-tutorial/doc/playground-overview.adoc:7]: Good point. I'll change back.

**Comment by simonjohnthompson** [comment]
Made all the changes asked for, and uploaded missing file

**Comment by simonjohnthompson** [comment]
Updated section 6 of the tutorial to new video links.

**Pull request merged**